scripts/plot-package-degrees.py

```diff
@@ -0,0 +1,26 @@
+"""
+Fetches all dependencies (first order) of each package and 
+plots the histogram of the number of dependencies (degree)
+
+
+"""
+import glob
+import pylab
+import json
+
+
```

Comment:
Please import `common`, and follow the style of the other plots (use seaborn, import matplotlib in the same way, ...). Use the same figure size (4,2).